apps/nuxt3-ssr/components/content/ContentBlock.vue

```diff
@@ -6,9 +6,7 @@ defineProps<{
 </script>
 
 <template>
-  <section
-    class="bg-white py-18 lg:px-12.5 px-5 text-gray-900 xl:rounded-3px last:rounded-b-50px last:mb-18 shadow-primary xl:border-b-0 border-b-[1px]"
-  >
```

Comment:
why are these styles removed ? , it seems to break the padding for the last section in a section list ( see about page, bottom of page in umcg theme for example)
<img width="1300" alt="Screenshot 2023-05-08 at 10 25 37" src="https://user-images.githubusercontent.com/47183404/236774709-6072552b-b7b7-4259-8f94-4ae256ba6aeb.png">